### ISSUE TITLE
[OfficeOnlineServerInstall and OfficeOnlineServerInstallLanguagePack] Installation hangs when installing from an UNC path

### ISSUE THREAD

**Issue by ykuijs** [dsccommunity/OfficeOnlineServerDsc#38]
#### Details of the scenario you tried and the problem that is occurring
When specifying an UNC path to install SharePoint, a Language Pack or CU, the installation hangs endlessly.

#### Verbose logs showing the problem
N/A

#### Suggested solution to the issue
I have investigated this issue and discovered that Start-Process gives a security warning when a UNC path is used. This can be resolved by adding the used servername to the IE Local Intranet Zone:
https://www.reddit.com/r/PowerShell/comments/7d49z3/start_script_from_uncpath_without_security_warning/
https://technochat.in/2008/12/disable-unc-path-security-warning-from-group-policy/

This can be resolved via PowerShell by executing:
```powershell
$null = New-Item -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\Domains\$serverName"
Set-ItemProperty -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\Domains\$serverName" -Name file -Value 1 -Type DWord
$null = New-Item -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\EscDomains\$serverName"
Set-ItemProperty -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\EscDomains\$serverName" -Name file -Value 1 -Type DWord
```

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
OfficeOnlineServerInstall Install_OOS_Binaries
{
    Path      = "\\server\install\oos\setup.exe"
    Ensure    = "Present"
}
```

#### The operating system the target node is running
All

#### Version and build of PowerShell the target node is running
All

#### Version of the DSC module that was used ('dev' if using current dev branch)
dev

**Comment by ykuijs** [comment]
Working on implementing this

**Comment by ykuijs** [comment]
Resolved in PR #39
